src/applications/mhv-secure-messaging/containers/App.jsx

```diff
@@ -143,10 +143,11 @@ const App = ({ isPilot }) => {
     <RequiredLoginView
       user={user}
       serviceRequired={[backendServices.MESSAGING]}
+      verify
     >
       {user.login.currentlyLoggedIn &&
       !userServices.includes(backendServices.MESSAGING) ? (
-        window.location.replace('/health-care/secure-messaging')
+        window.location.replace('/my-health')
```

Comment:
Why not use the same Paths constant here?

Reply:
I only noticed the `Paths` constant is in the e2e tests directory! Not sure why they weren't used here. Maybe I'll do a follow-up PR